commit 27a1a1e2e47d6f12a784413c194a94b6c0d7fdcb
Author: Carlos Santa <carlos.santa@intel.com>
Date:   Thu Oct 26 15:01:27 2023 -0700

    drm/xe: stringify the argument to avoid potential vulnerability
    
    This error gets printed inside a sandbox with warnings turned on.
    
    /mnt/host/source/src/third_party/kernel/v5.15/drivers/
    gpu/drm/xe/xe_gt_idle_sysfs.c:87:26: error: format string is
    not a string literal (potentially insecure) [-Werror,-Wformat-security]
            return sysfs_emit(buff, gtidle->name);
                                    ^~~~~~~~~~~~
    /mnt/host/source/src/third_party/kernel/v5.15/drivers/
    gpu/drm/xe/xe_gt_idle_sysfs.c:87:26: note: treat the string
    as an argument to avoid this
            return sysfs_emit(buff, gtidle->name);
                                    ^
                                    "%s",
    1 error generated.
    
    CC: Rodrigo Vivi <rodrigo.vivi@intel.com>
    Signed-off-by: Carlos Santa <carlos.santa@intel.com>
    Reviewed-by: Rodrigo Vivi <rodrigo.vivi@intel.com>
    Signed-off-by: Rodrigo Vivi <rodrigo.vivi@intel.com>

diff --git a/drivers/gpu/drm/xe/xe_gt_idle_sysfs.c b/drivers/gpu/drm/xe/xe_gt_idle_sysfs.c
index 7238e96a116c..8df9840811cd 100644
--- a/drivers/gpu/drm/xe/xe_gt_idle_sysfs.c
+++ b/drivers/gpu/drm/xe/xe_gt_idle_sysfs.c
@@ -85,7 +85,7 @@ static ssize_t name_show(struct device *dev,
 {
 	struct xe_gt_idle *gtidle = dev_to_gtidle(dev);
 
-	return sysfs_emit(buff, gtidle->name);
+	return sysfs_emit(buff, "%s\n", gtidle->name);
 }
 static DEVICE_ATTR_RO(name);